extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -511,10 +510,10 @@ export async function promptAndUpdateConnectionStringSetting(projectUri: vscode.
 /**
  * Prompts the user to include password in the connection string and updates the connection string based on user input
  * @param connectionInfo connection info from the connection profile user selected
- * @param localSettingsPath path to the local.settings.json file
+ * @param localSettingsPath path to the local.settings.json file (undefined when creating a new azure function project)
```

Comment:
The way this is worded makes me think that the parameter is required - it would just be undefined. In this case I would suggest just phrasing it that this field is optional instead